test/e2e/index.test.ts

```diff
@@ -129,4 +132,48 @@ describe('e2e', () => {
       await expect(screen.getByTestId('three')).toHaveFocus();
     });
   });
+
+  describe('<NavigationMenu />', () => {
```

Comment:
This test can be added to our unit test suite. We run it in the browser using Vitest's browser mode (using Playwright under the hood).